### ISSUE TITLE
Make the POC ConfigsManager as the new ConfigsManager Step 2

### ISSUE THREAD

**Issue by rider-cantuaria-eng** [SimplyVC/panic#334]
### Technical Story

The alerter needs to be notified whenever there is a configuration change.

### Description

Since we are now storing the configurations inside the database we must be able to detect a configuration change by listening to database events. This means that we need to migrate the POC logic that @simplyrider has developed and replace the old `ConfigsManager`

### Requirements

- [ ] Connect the new `ConfigsManager` with the `HEALTH_CHECK_EXCHANGE` so that the `ConfigsManager` can respond to heartbeats
- [ ] Check if the Slack Commands Handler requires changes in order to process heartbeats from the `ConfigsManager`
- [ ] Check if the Telegram Commands Handler requires changes in order to process heartbeats from the `ConfigsManager`

**Extra attention should be put towards the following**:

- On startup we need to send all configurations from the database.
- Error handling should be implemented similarly to the old `ConfigsManager`
- Proper thread handling of the newly created threads.
- Dynamic RabbitMQ routing keys based on configs.6

### Blocked by

#147 

### Acceptance criteria

**Given**:  Any Substrate configuration change
**Then**: The `ConfigsManager` sends the correct configuration update with the correct routing key

**Given**:  Any Cosmos configuration change
**Then**: The `ConfigsManager` sends the correct configuration update with the correct routing key

**Given**:  Any Chainlink configuration change
**Then**: The `ConfigsManager` sends the correct configuration update with the correct routing key

**Given**:  Any General configuration change
**Then**: The `ConfigsManager` sends the correct configuration update with the correct routing key

**Comment by itsciccio** [comment]
To be merged into https://github.com/SimplyVC/panic/issues/147
